### DISCUSSION TITLE
Conditional formatting for Formatted Value Display

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#9038] [Feature Requests]
It would be nice to set up conditional rules for formatting the values of the Formatted Value Display. Especially for integers, you could say it's RED from 0-30, YELLOW from 31-70, and GREEN from 71-100.

**Comment by haniefss** [comment]
> It would be nice to set up conditional rules for formatting the values of the Formatted Value Display. Especially for integers, you could say it's RED from 0-30, YELLOW from 31-70, and GREEN from 71-100.

It also would be nice to have a `higher_is_better` kind of condition to switch format used. Using your example, for some metrics a lower value might be desirable hence we can switch GREEN color for 0-30.

**Reply by azrikahar** [reply]
Hi, this was actually merged and implemented in [v9.3.0](https://github.com/directus/directus/releases/tag/v9.3.0) thanks to #9773! You'll be able to set conditional rules using it, and even do the exact logic that you described. 👍
